data/json/items/armor.json

```diff
@@ -6289,10 +6290,10 @@
         "id" : "fireman_belt",
         "name" : "firefighter belt",
         "volume" : 2,
-        "material_thickness" : 0,
+        "material_thickness" : 2,
```

Comment:
`material_thickness` is only for armor calculations. Firefighter belt has 0 coverage, meaning it will not work as armor.